qless/workers/forking.py

```diff
@@ -25,7 +30,7 @@ def __init__(self, *args, **kwargs):
 
     def stop(self, sig=signal.SIGINT):
         '''Stop all the workers, and then wait for them'''
-        for cpid in self.sandboxes.keys():
+        for cpid in list(self.sandboxes):
```

Comment:
You can drop this `list()` invocation. The default is to iterate over keys in both Python 2 & 3.